### ISSUE TITLE
Internal class extender is using a global reference to a closure variable

### ISSUE THREAD

**Issue by orizerah98** [jdalrymple/gitbeaker#1399]
Receiving 404 when using GroupMembers.all(groupId);
The same group id works for Groups.show(groupId);
When performing a manual GET request to https://gitlab.com/api/v4/groups/groupid/members/all I'm getting a valid response.
```
const gitlab = Gitlab({token: "TOKEN"})
gitlab.GroupMembers.all(groupId); // Throws an error.

await axios.get("https://gitlab.com/api/v4/groups/10450319/members", { headers: { "private-token": ACCESS_TOKEN },}); // Works fine
```

I've noticed that it tries to request the api without the `/groups/` section.

**Comment by jdalrymple** [comment]
Il look!

**Comment by jdalrymple** [comment]
Digging a bit deeper, seems that my bundler function has a bug in it. For the time being, you can use a direct import without a problem.

```js
import { GroupMembers } from '@gitbeaker/node';
const members = new GroupMembers({token: "TOKEN"})
members.all(groupId);
```

**Comment by jdalrymple** [comment]
For some reason, when using the Gitlab bundle, every serviceOptions [here](https://github.com/jdalrymple/gitbeaker/blob/master/packages/gitbeaker-requester-utils/src/RequesterUtils.ts#L84) is always a Wikis object :thinking: Still looking!

**Comment by jdalrymple** [comment]
I think the problem is derived from the reassignment of the [closure variable](https://medium.com/front-end-weekly/js-dont-reassign-closure-variables-ae6cf1083eb). Going to create a unconnected object to pass to the requester and see if that helps.

**Comment by jdalrymple** [comment]
That didnt work, for some reason the wrong information is being passed :thinking:

**Comment by jdalrymple** [comment]
Yup defintely seems like a closure problem i must have overlooked. With testing ive found that the closure function parameter always references the last service in the GitlabBundle (wikis in this case). Ill have to look into how to refactor to fix this. Any suggestions are welcome. :nerd_face:  Id like to know where i went wrong!

**Comment by jdalrymple** [comment]
This casues 404 when using GroupMembers.all() for example.

Verified that behaviour is for all bundles in the node package

**Comment by jdalrymple** [comment]
Doesnt seem related to the bundling, but the createRequesterFn. Still investigating

**Comment by jdalrymple** [comment]
I've managed to create a unit test to recreate the problem everytime. Still need to figure out the source of the error though

**Comment by jdalrymple** [comment]
haha, wow. When you see what the actual error was :facepalm: . Pushing up the fix now!

**Comment by orizerah98** [comment]
Thank you very much!

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`28.0.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/28.0.0) :rocket:
